README.md

```diff
@@ -137,6 +137,8 @@ You can do the same with the classes ClusterWorker, FixedClusterPool and Dynamic
 **See examples folder for more details (in particular if you want to use a pool for [multiple functions](./examples/multiFunctionExample.js)).**  
 **Now TypeScript is also supported, find how to use it into the example folder**.
 
+Remember that workers can only send and receive serializable (JSON) data.
```

Comment:
I am not sure that this is totally accurate based on node js doc, I need more time to look into that.

Reply:
At least it's a placeholder / reminder for us to improve the readme on this point ^^
jerome already moved forward :shrug: 
But I don't think that this is a problem

Reply:
No problem we can always be more detailed if we figure out that is needed :) it is ok